docs/source/configuration/settings-reference.md

```diff
@@ -200,8 +200,8 @@ controlpanels
 
     The group can be one of the default groups 'General', 'Content', 'Security', 'Add-on Configuration', 'Users and Groups' or a custom group.
 
-filterControlPanelsSchema
-    A schema factory for a control panel. It is used internally, to tweak the schemas provided by the controlpanel endpoint, to make them fit for Volto.
```

Comment:
I'm not sure what version number this will get released into, so I put the next unreleased version number for now. It might need to be updated when it is actually released.

````suggestion
    ```{deprecated} Volto 16.20.8
    `filterControlPanelsSchema` was renamed to `unwantedControlPanelsFields` in Volto 16.20.8.
    ```
````